DifferenceKit.podspec

```diff
@@ -14,6 +14,8 @@ Pod::Spec.new do |spec|
   spec.requires_arc = true
   spec.default_subspecs = 'Core', 'UIExtensions'
 
+  spec.swift_version = '4.1'
```

Comment:
Can we use variable to ensure the same Swift version in the main spec and sub spec?

```ruby
swift_version = '4.1'
spec.swift_version = swift_version

subspec.swift_version = swift_version
```